src/chai-form.ts

```diff
@@ -64,19 +75,22 @@ export class ChaiForm extends LitElement {
     if (window.location.hostname.includes('correirabros.com')) {
       this.overwrittenFlowType = 'correirabros.com';
     }
-    if (localStorage.getItem("chai-hotfix-version") === null || localStorage.getItem("chai-hotfix-version") === '1') {
+    if (
+      localStorage.getItem('chai-hotfix-version') === null ||
+      localStorage.getItem('chai-hotfix-version') === '1'
+    ) {
       localStorage.removeItem('chai-flowInstanceId');
       localStorage.setItem('chai-hotfix-version', '2');
     }
 
-    this.addEventListener(
-      'chai-fieldchanged',
-      this.handleFieldChange as (e: Event) => void
-    );
-    this.addEventListener(
-      'chai-fieldinit',
-      this.handleFieldInit as (e: Event) => void
-    );
+    // this.addEventListener(
+    //   'chai-fieldchanged',
+    //   this.handleFieldChange as (e: Event) => void
+    // );
+    // this.addEventListener(
+    //   'chai-fieldinit',
+    //   this.handleFieldInit as (e: Event) => void
+    // );
```

Comment:
Why did you comment out this existing functionality? You broke the form's behavior. 😲